packages/styles/scss/components/masthead/_masthead.scss

```
@@ -305,6 +313,102 @@ $search-transition-timing: 95ms;
     }
   }
 
+  .#{$prefix}--header__nav-caret-left,
+  .#{$prefix}--header__nav-caret-right {
+    width: rem(40px);
+    height: 100%;
+    display: none;
+    position: relative;
+    background: none;
+    border: none;
+    > {
+      svg {
+        position: relative;
+        fill: currentColor;
+        top: 2px;
```

An alternative is using `display: flex; justify-content: center; align-items: center` for the parent `<button>`:

```suggestion
        position: absolute;
        fill: currentColor;
        top: 50%;
        left: 50%;
        transform: translateX(-50%) translateY(-50%);
```